app/views/leads/_lead.json.jbuilder

```diff
@@ -0,0 +1,2 @@
+json.extract! lead, :id, :parent_id, :otu_id, :text, :origin_label, :description, :redirect_id, :link_out, :link_out_text, :position, :is_public, :project_id, :created_by_id, :updated_by_id, :created_at, :updated_at
```

Comment:
We've been calling these `_attributes.json.jbuilder` before this pattern came out (we anticiapted this convention before the convention happened in essence).  Rename it that for now to keep that consistency across our models.